OrbitGrpcProtos/capture.proto

```diff
@@ -83,6 +83,22 @@ message InternedString {
   string intern = 2;
 }
 
+message InternedTracepointInfo {
+  uint64 key = 1;
+  TracepointInfo intern = 2;
+}
+
+message TracepointEvent {
+  int32 pid = 1;
+  int32 tid = 2;
+  int64 time = 3;
+  int32 cpu = 4;
+  oneof tracepoint_info_or_key {
```

Comment:
I think the oneof should be replaced by 
`uint64 tracepoint_info_key = 5;`

We'll send all the InternedTracepointInfo's independent of this. Let's discuss on Monday ...

Reply:
Of course! I am really curious about how I could do this. I am looking forward to discuss this on Monday. Thank you!